common/source/docs/common-mamba405-mk2.rst

```diff
@@ -0,0 +1,172 @@
+================================
+Mamba F405 MK2 Flight Controller
+================================
+
+.. image:: ../../../images/mamba_f405mk2_board.jpg
+     :target: ../_images/mamba_f405mk2_board.jpg
+
+The Mamba F405 MK2 is a flight controller produced by
+`Diatone <https://www.diatone.us>`__.
+
+
+Features
+========
+
+-  STM32F405RGT6 microcontroller
+-  MPU6000 IMU
+-  AT7456E OSD
+-  4 UARTs
+-  4 PWM outputs
+
+Where to buy
+============
+
+Available from multiple retailers and directly from the vendor at `diatone.us <https://www.diatone.us/products/mamba-f405-flight-controller-mk2>`__
+
+
+Pinout
+======
+
+.. figure:: ../../../images/mamba_f405mk2_pinout.png
+   :target: ../_images/mamba_f405mk2_pinout.png
+
+   Mamba F405 MK2 Board
+
+UART Mapping
+============
+
+======= ========== ===================
+Name    Pin labels Function
+======= ========== ===================
+SERIAL0            USB
+SERIAL1 PPM / SBUS RC Input / UART1 RX
+SERIAL3 TX3 / RX3  UART3 (Telem1)
+SERIAL6 TX6 / RX6  UART6 (GPS)
+======= ========== ===================
+
+It is possible to use PPM pin as SERIAL1 TX instead of RC input by setting :ref:`BRD_ALT_CONFIG<BRD_ALT_CONFIG>` to 1.
```

Comment:
incorrect....and its the RX not TX pin....I would just not mention ALT config until its gets fixed to also disable the inverter output driving the PPM pin from the SBUS pin

Reply:
and the hwdef has UART1 defaulted to TELEM1(which does not matter) and UART3 defaulted to GPS1, UART6 is USER2, so the pinout diagram needs changing also...its for a BETAFLIGHT setup, not AP